bundle/src/main/java/com/adobe/acs/commons/oak/impl/EnsureOakIndexManagerImpl.java

```diff
@@ -92,8 +94,6 @@ public class EnsureOakIndexManagerImpl extends AnnotatedStandardMBean implements
         )
         String[] properties_ignore() default {};
 
-        String webconsole_configurationFactory_nameHint() default "Additional Ignore properties: {properties.ignore}";
```

Comment:
This should be kept as it eases maintenance in the UI.

Reply:
done